system/nagios-status-check/README.md

```diff
@@ -0,0 +1,70 @@
+
+On the agent side, you need to send your metrics with the [telegraf exec plugin][https://docs.signalfx.com/en/latest/integrations/agent/monitors/telegraf-exec.html] which includes a [parser for nagios](https://github.com/influxdata/telegraf/blob/master/plugins/parsers/nagios/parser.go). The metric is named `nagios_state.state`, you need to add an `extraDimensions` to your monitor in order to be able to differantiate multiple script states.
+
+Here is an example : 
+
+```yaml
+- type: telegraf/exec
+  intervalSeconds: 30
+  command: /usr/local/bin/scripts/check-ipmitool.pl
+  extraDimensions:
+    script: check_ipmitool
+  telegrafParser:
+    dataFormat: nagios
+```
+
+Ideally, use an `extraDimensions` which allows to easily identify the executed script.
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| detectors\_disabled | Disable all detectors in this module | bool | `"false"` | no |
+| environment | Infrastructure environment | string | n/a | yes |
+| filter\_custom\_excludes | List of tags to exclude when custom filtering is used | list | `[]` | no |
+| filter\_custom\_includes | List of tags to include when custom filtering is used | list | `[]` | no |
+| notifications | Notification recipients list for every detectors | list | n/a | yes |
+| prefixes | Prefixes list to prepend between brackets on every monitors names before environment | list | `[]` | no |
+| status\_check\_aggregation\_function | Aggregation function and group by for status\_check detector \(i.e. ".mean\(by=\['host'\]\)."\) | string | `""` | no |
+| status\_check\_disabled | Disable all alerting rules for status\_check detector | bool | `"null"` | no |
+| status\_check\_disabled\_critical | Disable critical alerting rule for status\_check detector | bool | `"null"` | no |
+| status\_check\_disabled\_warning | Disable warning alerting rule for status\_check detector | bool | `"null"` | no |
+| status\_check\_notifications | Notification recipients list for every alerting rules of status\_check detector | list | `[]` | no |
+| status\_check\_notifications\_critical | Notification recipients list for critical alerting rule of status\_check detector | list | `[]` | no |
+| status\_check\_notifications\_warning | Notification recipients list for warning alerting rule of status\_check detector | list | `[]` | no |
+| status\_check\_transformation\_function | Transformation function for status\_check detector \(mean, min, max\) | string | `"min"` | no |
+| status\_check\_transformation\_window | Transformation window for status\_check detector \(i.e. 5m, 20m, 1h, 1d\) | string | `"15m"` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| status\_check\_id | id for detector status\_check |
+
+## Related documentation
+
+[Official documentation for the telegraf exec plugin](https://docs.signalfx.com/en/latest/integrations/agent/monitors/telegraf-exec.html)
```

Comment:
you should probable add newline or create a list, every docs will be on the same line which is difficult to read